File: 062. Unique Paths.swift
class Solution {
    func uniquePaths(_ m: Int, _ n: Int) -> Int {
        var map = Array(repeating: Array(repeating: 1, count: n), count: m)
        
        for i in 1 ..< m {
            for j in 1 ..< n {
                map[i][j] = map[i-1][j] + map[i][j-1]
            }
        }
        
        return map[m-1][n-1]
    }
}

File: LeetCode/062.Unique Paths.py
class Solution(object):
    def uniquePaths(self, m, n):
        """
        :type m: int
        :type n: int
        :rtype: int
        """
        matrix = [[1 for x in range(n)] for y in range(m)]
        for i in range(1, m):
             for j in range(1, n):
                matrix[i][j] = matrix[i - 1][j] + matrix[i][j - 1]
        return matrix[m - 1][n - 1]
